### ISSUE TITLE
Decorative icons in menu are focusable

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#1454]
### Description of problem
Decorative icons in menu are focusable. Since there's no description of the icon, the focus is empty. A non-sighted person will not know what it is they are focused on.

### Expected behavior
Icons should be labelled as decorative so that they aren't focusable. 

### Steps to reproduce
- Turn on Talkback
- go to menu
- scroll through the menu

### Screenshots and/or log output
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBblNRIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--272c60f311f21375b263f5283a231ec0a0d24228/20230912_094905.png)

### Environment
- [X] Occurs on Android
- [ ] Occurs on iOS

Build #: 1193

Android Device Model: OnePlus 6

iOS Device Model: 

### Severity
- [ ] High
- [ ] Medium
- [X] Low

**Comment by knguyenBC** [comment]
Checked. looks good.
